website/server/controllers/api-v3/members.js

```diff
@@ -413,7 +413,10 @@ api.getChallengeMemberProgress = {
     // manually call toJSON with minimize: true so empty paths aren't returned
     let response = member.toJSON({minimize: true});
     delete response.challenges;
-    response.tasks = chalTasks.map(chalTask => chalTask.toJSON({minimize: true}));
+    response.tasks = chalTasks.map(chalTask => {
```

Comment:
I think it's easier to just edit line 410 here that already contains some data to remove

Reply:
I'd rather not because that would remove the value from the response and any apis expect the existence of the checklists would break. So returning an empty array is safer.

Reply:
👍